Makefile

```diff
@@ -81,17 +80,17 @@ local/bin/python3.7:
 
 .PHONY: lint
 lint: .venv/build.timestamp
-		$(YAPF_CMD) -i --style .style.yapf $(PYTHON_FILES)
+	$(YAPF_CMD) -p -i --style .style.yapf $(PYTHON_FILES)
+	$(PYLINT_CMD) $(PYTHON_FILES)
 
 # Serve targets. Using these will run the application on your local machine. You can either serve with a wsgi front (like it would be within the container), or without.
 .PHONY: serve
 serve: .venv/build.timestamp
-		FLASK_APP=service_launcher FLASK_DEBUG=1 ${FLASK_CMD} run --host=0.0.0.0 --port=${HTTP_PORT}
+	FLASK_APP=service_launcher FLASK_DEBUG=1 ${FLASK_CMD} run --host=0.0.0.0 --port=${HTTP_PORT}
```

Comment:
Use `INSTALL_DIR`

Reply:
Not sure if I understand what you mean, but yeah the make targe .venv/build.timestamp should use the variable INSTALL_DIR. This can be also done in another PR.